hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ConfigurationService.java

```diff
@@ -86,13 +87,29 @@
      */
     ConfigUpdateResult update(Config newConfig);
 
+    /**
+     * Starts a configuration process asynchronously. Updates the configuration with the given configuration. Updating means
```

Comment:
Mİnor:

Maybe `Starts a configuration process` -> `Starts a configuration update process`?

Also, can you re-order those two so, instead of `update(Config), updateAsync(String), updateAsync(), update()`, we would have `update(Config), update(), updateAsync(String), updateAsync()`

Reply:
Sure, done.